dask_ml/model_selection/_hyperband.py

```diff
@@ -0,0 +1,325 @@
+import math
+import numpy as np
+import toolz
+from sklearn.base import clone
+from sklearn.model_selection import ParameterSampler
+from sklearn.utils import check_random_state
+
+from tornado import gen
+from toolz import groupby
+from dask.distributed import as_completed, default_client, futures_of
+from dask_searchcv.model_selection import DaskBaseSearchCV
+import dask.array as da
+from ..datasets import make_classification
+from ..wrappers import Incremental
+from ._split import train_test_split
+from distributed.metrics import time
+from distributed import Future
+from copy import deepcopy
```

Comment:
I believe that the convention for imports is 

1.  standard libraries
2.  third party packages
3.  relative imports within this project

All groups separated by a line, all groups alphabetized within each group, so

```python
import copy
import math

import dask.array as da
from dask.distributed import Future, as_completed, ...

from ..datasets import make_classification
from ..wrappers import Incremental
```